resources/views/master.blade.php

```diff
@@ -10,11 +10,13 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
         integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="shortcut icon" href="{{ asset('favicon.ico') }}">
+    <link rel="stylesheet" href="{{asset('css/app.css')}}">   <!-- BOOTSTRAP -->
```

Comment:
we are using bootstrap in two ways:
- CND (link)
- Downloaded (asset)

Use only one